orders/src/main/java/com/lambdaschool/orders/model/Customers.java

```diff
@@ -0,0 +1,190 @@
+package com.lambdaschool.orders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "customers")
+public class Customers
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long custcode;
+
+    @Column(nullable = false)
+    private String custname;
+
+    private String custcity;
+    private String workingarea;
+    private String custcountry;
+    private String grade;
+    private double openingamt;
+    private double receiveamt;
+    private double paymentamt;
+    private double outstandingamt;
+    private String phone;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "agentcode",
+            nullable = false)
+    @JsonIgnoreProperties({"customers", "hibernateLazyInitializer"})
+    private Agents agent;
+
+    @OneToMany(mappedBy = "customer",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true)
+    @JsonIgnoreProperties("customers")
+    private List<Orders> orders = new ArrayList<>();
+
+    public Customers()
```

Comment:
Line 39 - Use an array of strings, and make sure to both correct the property typo (singular **"customer"**, not plural "customers") and include the property "agent" as well to avoid recursion (with your current setup, the `customer` and `agent` properties will still not print because of your use of `@JsonIgnore` in the `Orders.java` model, but using `@JsonIgnore` shouldn't be necessary and will actually prevent you from printing some order information if you were to use the `.findAll` methods on it directly, for instance).

To use the array of strings: **`@JsonIgnoreProperties({"customer", "agent"})`**